app/src/utils/Utils.ts

```diff
@@ -79,3 +80,22 @@ export const getFormattedAmount = (amount: number): string => {
       .replace(/\B(?=(\d{3})+(?!\d))/g, ',')
   );
 };
+
+/**
+ * Log the user out, redirecting them to the logout page.
```

Comment:
question: should this say
```suggestion
 * Log the user out, redirecting them to the login page.
```

Reply:
Fixed